### PR TITLE
fetch generated png file support though http proxy

### PR DESCRIPTION
**Pull request by joyhope**
I added a fetch png file with proxy support.  The default file box does not support proxy, so I use Axios to save the file and recreate file box.  It caused extra disk i/o. 

It is my first time writing typescript, I am pretty sure lots of code needs to improve.

### CODE REVIEW

**Comment by novicezk** [comment]
HTTP_PROXY不配置或为空时，应使用原有逻辑，有些vpn不需要proxy的 @joyhope

**Comment by joyhope** [comment]
应该默认的不配置, 和空 就是老流程 . 没修改.

File: src/main.ts [hunk 1]
```diff
@@ -73,13 +74,25 @@ app.post("/notify", async (req: Request, res: Response): Promise<Response> => {
...
       } else {
         const taskId = req.body.id;
         const prompt = req.body.prompt;
         await room.say(`@${userName} \n🎨 ${action == 'IMAGINE' ? '绘图' : '变换'}成功，用时 ${displayMilliseconds(time)}\n✨ Prompt: ${prompt}\n📨 任务ID: ${taskId}\n🪄 放大 U1～U4 ，变换 V1～V4\n✏️ 使用[/up 任务ID 操作]\n/up ${taskId} U1`);
+
+        /*
         const image = FileBox.fromUrl(req.body.imageUrl);
+        */
+        console.log(`${req.body.imageUrl}`);
+        const savedFileName = await downloadImage(req.body.imageUrl);
```
Comment (novicezk) [inline src/main.ts]: 没有配置proxy时，应该不需要下载文件到本地，直接FileBox.fromUrl(req.body.imageUrl)就可以了

File: src/main.ts [hunk 2]
```diff
@@ -73,13 +75,31 @@ app.post("/notify", async (req: Request, res: Response): Promise<Response> => {
       const time = req.body.finishTime - req.body.submitTime;
       if (action == 'UPSCALE') {
         await room.say(`@${userName} \n🎨 图片放大，用时: ${displayMilliseconds(time)}\n✨ ${description}`);
-        const image = FileBox.fromUrl(req.body.imageUrl);
+
+        let image:FileBox;
+
+        if (config.httpProxy !== "") {
```
Comment (novicezk) [inline src/main.ts]: 这里是不是写反了，不等于空时走downloadImage

**Pull request merged**